src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/mapper/letterdetails/generalletter/GeneralLetterDetailsMapper.java

```diff
@@ -60,27 +67,17 @@ public DocumentTemplateDetails buildDocumentTemplateDetails(FinremCaseDetails ca
     }
 
     private Addressee getAddressee(FinremCaseDetails caseDetails) {
+        FinremCaseData data = caseDetails.getData();
+        String generalLetterAddressee = data.getGeneralLetterWrapper().getGeneralLetterAddressee().getValue().getCode();
+        boolean recipientResideOutsideOfUK = postalService.isRecipientResideOutsideOfUK(data, generalLetterAddressee);
         return Addressee.builder()
                 .name(getRecipientName(caseDetails))
-                .formattedAddress(formatAddressForLetterPrinting(getRecipientAddress(caseDetails)))
+                .formattedAddress(formatAddressForLetterPrinting(getRecipientAddress(caseDetails), recipientResideOutsideOfUK))
                 .build();
     }
 
-    public static String formatAddressForLetterPrinting(Address address) {
-        return formatAddressForLetterPrinting(new ObjectMapper().convertValue(address, Map.class));
-    }
-
-    private static String formatAddressForLetterPrinting(Map<String, Object> address) {
-        if (address != null) {
-            return Stream.of("AddressLine1", "AddressLine2", "AddressLine3", "County", "PostTown", "PostCode")
-                .map(address::get)
-                .filter(Objects::nonNull)
-                .map(Object::toString)
-                .filter(StringUtils::isNotEmpty)
-                .filter(s -> !s.equals("null"))
-                .collect(Collectors.joining("\n"));
-        }
-        return "";
+    public String formatAddressForLetterPrinting(Address address, boolean isInternational) {
+        return documentHelper.formatAddressForLetterPrinting(new ObjectMapper().convertValue(address, Map.class), isInternational);
```

Comment:
There is no need to create a new ObjectMapper; this class already has one

Reply:
done